pkgs/applications/science/chemistry/avogadro/default.nix

```diff
@@ -1,11 +1,13 @@
-{ stdenv, fetchurl, cmake, qt4, zlib, eigen, openbabel, pkgconfig, mesa, libX11, doxygen }:
+{ stdenv, fetchFromGitHub, cmake, qt4, zlib, eigen, openbabel, pkgconfig, mesa, libX11, doxygen }:
 
 stdenv.mkDerivation rec {
-  name = "avogadro-1.1.1";
+  name = "avogadro";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/avogadro/${name}.tar.bz2";
-    sha256 = "050ag9p4vg7jg8hj1wqfv7lsm6ar2isxjw2vw85s49vsl7g7nvzy";
+  src = fetchFromGitHub {
+    owner = "cryos";
+    repo = "avogadro";
+    rev = "92e20f0072c51badd0849eb354b5cf4bfd05e6a3";
```

Comment:
you can specify a tag as revision as well

Reply:
Should I change it?

Reply:
just personal taste, but pinning a package to a git tag explicitly shows me that this no random/unstable stuff from GitHub :-)